UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Traitor.cs

```diff
@@ -21,6 +23,8 @@ public override void AfterSpawn(ConnectedPlayer player)
 		{
 			if (player.GameObject.TryGetComponent<AiPlayer>(out var aiPlayer))
 			{
+				aiPlayer.IsMalf = true;
+				AddObjective(aiTraitorObjective);
```

Comment:
Yeah can you make this a percentage based chance to add

Reply:
25% think would be fine

Reply:
There no reason for it to be chance based and I've intentionally made it so that AIs always get the "Prevent all lifeforms from escaping" objective.

Reply:
> There no reason for it to be chance based and I've intentionally made it so that AIs always get the "Prevent all lifeforms from escaping" objective.

Why?

Reply:
Seems to make all traitor AI rounds very similar if they have to prevent all players from leaving and will just make it so the Ai plasma floods all the time.

Reply:
Now correct me if I'm wrong. But that has been always the malf AI objective whenever you played that.
Sure, Extra objectives existed but I don't want to follow the /TG/ philosophy and design in everything since I'm already against the idea of removing malf AI as a gamemode.

I don't see how it would be repetitive when all game modes except normal traitor follow the same logic of "you have an objective for your role specifically, do it."; If we began looking at everything with that same lens then all SS13 game modes are repetitive.

Reply:
SS13 game modes are repetitive. Sure which is a fundamental problem for some gamemodes like cult, but with traitor you get variation based on the objectives being different each time you play. When you force the AI to always prevent everyone from escaping you make it repetitive for traitor Ai's

Reply:
I think I agree with ThatDaniel. Variety is the spice of life.

Reply:
is there a goal to stay on the station alive no matter what?

Reply:
Alright will add the chance to it.
And I think there is? forget it's name